### ISSUE TITLE
lost-center no-flex flag seems to be broken in ^8.1.0 (working in 8.0.0)

### ISSUE THREAD

**Issue by cbravo** [peterramsing/lost#404]
**Is this a feature request or a bug report?**

Bug Report

**What is the current behavior?**
```
@lost flexbox flex;

.root {
  lost-center: 600px no-flex;
}
```
CSS for flexbox still being a applied to div Despite having the no-flex flag explicitly set on the lost-center property

**If it's a bug please provide the steps to reproduce it and maybe some code samples.**
I am using 
"webpack": "3.8.1",
"webpack-dev-server": "2.9.4",

**What is the desired behavior?**
it should not add the flex properties to the class if the no-flex flag is present!

**What version of LostGrid, browser and browser version are affected by this issue? Did this happen in previous versions?**

broken in ^8.1.0 (working in 8.0.0)

**Comment by peterramsing** [comment]
Hmm. Looking into this.

**Comment by peterramsing** [comment]
I'm not seeming to be able to reproduce this in isolation. Let me double check if there is a test case that covers this but there might be something else going on here.

**Comment by peterramsing** [comment]
Scratch that! I have a failing test. Weird. Let me see if I can get something patched up here.

**Comment by peterramsing** [comment]
Alright, I have a fix. Gonna slate it all together and see if I can get a patch out ASAP.

**Comment by peterramsing** [comment]
Closing this for #405.

Thanks, @cbravo for bringing this up.

**Comment by cbravo** [comment]
@peterramsing You are very welcome. Thank you for getting it sorted! Great work on this and I cannot express my appreciation enough for making this tool available to the world!

**Comment by peterramsing** [comment]
@cbravo Thanks. There's a lot of people who've made this a success. Feel free to offer any thoughts on the fix on the PR (#405) if you have any!

**Comment by peterramsing** [comment]
@cbravo This has been fixed and is released.
